api/BaseAudioContext.json

```diff
@@ -11,7 +11,7 @@
             "version_added": true
           },
           "edge": {
-            "version_added": "≤18"
+            "version_added": "12"
```

Comment:
https://mdn-bcd-collector.appspot.com/tests/api/BaseAudioContext returns false for Edge 18, but I understand you had to set this to "12" since some subfeatures were in Edge 12.

I think we should instead have a `partial_implementation` from Edge 12-18 with a note saying that while the BaseAudioContext interface itself wasn't supported, most of it members were on AudioContext. Since BaseAudioContext is an abstract interface it doesn't matter.

@ddbeck @Elchi3 for suggestions on note wording. We could use the same wording for other similar situations in https://github.com/mdn/browser-compat-data/issues/7517.

Reply:
Since I started work on the mixin business I wonder about how much documentation should sometimes hide weird facts. Here, I think, it might be another of those cases, where adding this fact to compat data or to docs is just confusing people and adds to the perception of the API surface as a highly fragile construct and the web being always incompatible due to many differences across engines.

Which is a long way of saying: Is this really that interesting and needs to be talked about? If the functionality is there and it is working, why do you think support is partial only?

Reply:
Ah, I now see this is about `BaseAudioContext` itself, and not its members. Still, it seems `partial_support` potentially alarms people about this being incompatible when it isn't really?

Reply:
Indeed this is only about what to do with the `api.BaseAudioContext` entry. Although abstract interfaces like `BaseAudioContext` or `SVGGeometryElement` do appear on the prototype chain and can be observed, I wouldn't say that their existence is especially interesting.

My concern is leaving some kind of trace of the fact that `BaseAudioContext` wasn't originally there, so that if in the future any tooling tests for its existence and suggests our data is wrong, we won't forget. So it's really notes for maintainers I want here.

That being said, I'd imagine two support statements, where the most recent has no notes at all, so this wouldn't end up looking especially scary in the resulting MDN tables.

Reply:
> That being said, I'd imagine two support statements, where the most recent has no notes at all, so this wouldn't end up looking especially scary in the resulting MDN tables.

This sounds reasonable to me. Can you do that here?

> So it's really notes for maintainers I want here.

`maintainer_notes` as a new BCD property then? I guess there is an issue for that? (/me dreams of JSON with comments, ah well).

Reply:
I couldn't find an open one, so I opened https://github.com/mdn/browser-compat-data/issues/9223

Reply:
For this PR, should we use a note in the interim?

Reply:
Yeah, I think a note for now and revisiting this when we do something about #9223 is the way to go.

Reply:
OK, so I suggest a range for Edge 12-18 with the note "The <code>BaseAudioContext</code> interface itself is not present, but many of the methods are available on the <code>AudioContext</code> interface." Then a range starting at Edge 79 with no notes.